### PR TITLE
fix: remove underline for logos

### PR DESCRIPTION
**Pull request by AemieJ**
In PR #452, the case for removal of underline from logos within docs and community was overlooked. With these changes, it is resolved.

### CODE REVIEW

File: antora-ui-camel/src/css/docs.css [hunk 1]
```diff
@@ -34,6 +34,11 @@
   margin: 1.5rem;
 }
 
+.section .logo a,
+.camel-project .logo a {
+  background: none;
+}
```
Comment (zregvart) [inline antora-ui-camel/src/css/docs.css:40]: I'm sure we'll forget to wrap the link with image in a `<div class="logo">`, I think we could use `:only-child` CSS selector so we don't have to, something like `a img:only-child` instead.

**Comment by aashnajena** [comment]
CSS for blog page also need to be fixed. `background : none` for `.doc li a` is causing categories to disappear. The cases for `a.category` need to be handled separately so that we don't see lines appearing on the category tags.

Screenshot from 2020-08-15 17-44-49

On another note, this is the view on Firefox and I'm not sure if the underlining looks very elegant. I like the picture that @zregvart  posted on #452 with light-gray dotted underline for links better than blue dashed strokes. Also, can we have some space between the text and the underline? I think this was discussed on #452 but I'm not sure what the conclusion was reached for that.

**Comment by AemieJ** [comment]
@aashnajena the gray dotted lines look highly clustered on the chrom browser whereas the dash looks structured, consistent and neat. We reached to the conclusion to not include a space between the text and underline.

File: antora-ui-camel/src/css/docs.css (hunk 1 above)
Comment (AemieJ) [reply antora-ui-camel/src/css/docs.css]: This doesn't work as I have to select the parent, though I have found the fix for it and done the required changes.

**Comment by AemieJ** [comment]
@zregvart @aashnajena let me know if you can find any errors, I will fix it. I think it's good to merge it if it's good to go as the changes of underline are already built on the website.

**Comment by aashnajena** [comment]
Can we also remove underlines from "Continue reading" links on Blog and "Read More" links on the front page? They're obvious without the underline as well and I don't really like how to underline extends beneath the " > " sign
Screenshot from 2020-08-16 14-53-06

File: antora-ui-camel/src/css/category.css [hunk 2]
```diff
@@ -24,6 +23,15 @@ li > a.category {
   line-height: 2;
 }
 
+a.category,
+li > a.category {
+  background-color: var(--color-camel-orange);
+  background-image: none;
+  background-position: 0;
+  background-repeat: no-repeat;
```
[the remark this answers is not in the document]
Comment (AemieJ) [reply antora-ui-camel/src/css/category.css:31]: Yes, as they have a background color I can't set the background to none for the categories.

[the remark this answers is not in the document]
Comment (aashnajena) [reply antora-ui-camel/src/css/category.css:31]: Okay, I tried removing those and it didn't make a difference so I thought it's not required. Check once, else we can merge this

[the remark this answers is not in the document]
Comment (AemieJ) [reply antora-ui-camel/src/css/category.css:31]: If you remove those lines, underlines will be present and if you set the background to none, it won't be visible. I checked it within the preview itself.

**Comment by zregvart** [comment]
I guess something went wrong with the rebase, I've created #468 to compensate.

**Comment by AemieJ** [comment]
@zregvart I observed that but there are some additional checks it has failed.

**Pull request closed without merging**